lib/iris/_lazy_data.py

```diff
@@ -95,6 +95,10 @@ def as_concrete_data(data, **kwargs):
     Returns:
         A NumPy `ndarray` or masked array.
 
+    .. note::
+        Specific dask options for computation are controlled by
+        :class:`iris.options.Parallel`.
```

Comment:
@dkillick This is stale, should be `iris.config.Parallel` or should it be `iris.config.parallel` ?